frontend/src/metabase/meta/Parameter.js

```diff
@@ -145,7 +238,7 @@ export function variableFilterForParameter(
 }
 
 function tagFilterForParameter(parameter: Parameter): TemplateTagFilter {
-  const [type, subtype] = parameter.type.split("/");
+  const [type, subtype] = splitType(parameter);
```

Comment:
I think this should include number params, but we probably also want to use subtype. A betweens filter can't be connected to a number template tag but single arity subtypes can.

Reply:
I ended up preventing usage of any operators besides `=` when mapping to tags. Do we need it now?

Reply:
That should work in most cases. Longer term, we probably want to allow other subtypes since the variable/tag might be used that way in the query. e.g. I might want to map a "starts with" parameter to a query with `column LIKE {{tag}} || '%'`. We can't know that's how `tag` is used, so the right behavior is probably to be too permissive and allow it if it could make sense.

Reply:
Agree that it'd be nice to allow this going forward but it requires some additional UI work on the native q side of things, so I'll keep it out of this PR for now.